backend/src/LeanCode.AppRating/MassTransitRegistrationConfigurationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using LeanCode.AppRating.Handlers;
+using MassTransit;
+
+namespace LeanCode.AppRating;
+
+public static class MassTransitRegistrationConfigurationExtensions
+{
+    public static void AddAppRatingConsumers<TUserId>(this IRegistrationConfigurator configurator)
+        where TUserId : notnull, IEquatable<TUserId>
+    {
+        configurator.AddConsumer(
+            typeof(SendEmailOnLowRateSubmittedEH<TUserId>),
+            typeof(AppRatingConsumerDefinition<SendEmailOnLowRateSubmittedEH<TUserId>>)
+        );
+    }
+}
+
+public class AppRatingConsumerDefinition<TConsumer> : ConsumerDefinition<TConsumer>
```

Comment:
I wonder, on one hand, we allow configuring too much (e.g. BCC, or default locale/culture), on the other, we set some things in stone and disallow any changes (consumer configuration). Where is the "ideal" line between ease of use and usability when it comes to configuring things?